rpcserver.go

```diff
@@ -1089,6 +1090,71 @@ func handleGetBlockHash(s *rpcServer, cmd interface{}, closeChan <-chan struct{}
 	return sha.String(), nil
 }
 
+// handleGetBlockHeader implements the getblockheader command.
+func handleGetBlockHeader(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	c := cmd.(*btcjson.GetBlockHeaderCmd)
+
+	sha, err := wire.NewShaHashFromStr(c.Hash)
+	if err != nil {
+		return nil, err
+	}
+
+	if *c.Verbose {
```

Comment:
The `Verbose` flag is optional, so this needs to be checked for nil before dereferencing it.

`if c.Verbose == nil || *c.Verbose {`